app/routers/dayview.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+
+from fastapi import APIRouter, Request
+from fastapi.templating import Jinja2Templates
+
+from app.dependencies import TEMPLATES_PATH
+
+
+templates = Jinja2Templates(directory=TEMPLATES_PATH)
+
+
+router = APIRouter()
+
+
+# inner class of the router, for the jinja page to process the json
+class Event:
```

Comment:
Is there any reason not to use the general `Event` object from the ORM?

Reply:
זה בעיקר בשביל הפונקציות של החישוב של הגריד. עדיף לפתוח מחלקה אחרת בשביל זה? או להשתמש בהן מחוץ למחלקה ולעבוד על אובייקט מהORM?

Reply:
So maybe we should call it `EventOnGrid`?

Reply:
השתמשתי בevent מהORM ושינתי את שם המחלקה ל Divattributes